<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>txtview</class>
 <widget class="QGroupBox" name="txtview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>read from text file</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>26</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="col0">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>80</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Column indexes (-1 - empty (zeros))</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="col1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="col2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>dY</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
